Block/SentryScript.php

```diff
@@ -3,10 +3,23 @@
 namespace JustBetter\Sentry\Block;
 
 use JustBetter\Sentry\Helper\Data as DataHelper;
+use JustBetter\Sentry\Helper\Version;
 use Magento\Framework\View\Element\Template;
 
 class SentryScript extends Template
 {
+    const CURRENT_VERSION = '5.17.0';
```

Comment:
This constant seems to be unused, could you please double check this and if it is unused. Remove it?

Reply:
It is used (phtml):
```
<script src="https://browser.sentry-cdn.com/<?= \JustBetter\Sentry\Block\SentryScript::CURRENT_VERSION;?>/bundle.min.js"
```

Reply:
I accidentally looked past it, it looks good 👍